pydra/engine/submitter.py

```diff
@@ -24,7 +25,7 @@ def __init__(self, plugin, runnable):
         else:
             raise Exception("plugin {} not available".format(self.plugin))
 
-        if hasattr(runnable, 'interface'):  # a node
+        if isinstance(runnable, Node):  # a node/task
```

Comment:
shouldn't this be BaseNode, i.e. any basenode derivative class is runnable?

Reply:
sure